categories.txt

```diff
@@ -9,8 +9,9 @@ Exploitation
 File Information
 Forensic
 Hex Editors
+IDA Plugins
 InnoSetup
-Java & Android
+Java and Android
```

Comment:
[nitpick] is the following possible?
```suggestion
Java/Android
```

@mandiant/flare-vm any preference between the two?

Reply:
Java/Android is possible as well, let me know which one is preferred

Reply:
I prefer `Java/Android` that is a bit shorter, but I have no strong opinion. So up to you if nobody else has an opinion. :wink:

Reply:
then and, cause it is not possible to create a folder with / in windows